src/components/config/config.ts

```diff
@@ -139,3 +139,19 @@ export function getConfiguredNamespace(): string | undefined {
 export function affectsUs(change: vscode.ConfigurationChangeEvent) {
     return change.affectsConfiguration(EXTENSION_CONFIG_KEY);
 }
+
+export function getEnabledLinters(): string[] {
+    const config = vscode.workspace.getConfiguration(EXTENSION_CONFIG_KEY);
+    return getEnabledLintersInternal(config['disable-lint'], config['use-linters']);
```

Comment:
These are new config settings - they should be described in `package.json::contibutions` and `readme`.

Reply:
done.